openapi/components/schemas/Cashier/CashierRequest.yaml

```
@@ -67,6 +67,41 @@ properties:
     description: Date and time at which the cashier request expires. The default expiration time is one hour from the time the request is created.
     type: string
     format: date-time
+  customPropertySetId:
+      description: ID of a custom property set to be applied for this request `propertiesSchema`.
```

nitpick: suggestion to avoid [passive voice](https://product.rebilly.dev/writing-style-guide/grammar-and-mechanics/tone-and-voice/voice/#passive-voice)
```suggestion
      description: ID of a custom property set to apply to the request `propertiesSchema`.
```